src/imagePane.js

```diff
@@ -45,7 +45,8 @@ module.exports = {
     return 'view'
   },
 
-  render: function (subject, myDocument) {
+  render: function (subject, context) {
+    const myDocument = context.dom
```

Comment:
`dom` isn't actually the `HTMLDocument` object, right? i.e. `myDomElement` would be more descriptive of what it actually is?

Reply:
`dom` is the actual `HTMLDocument`, yes.

Reply:
Huh, if that's the case, then it's a bit weird to pass it around in the context, right, given that it's available globally anyway?

Reply:
Not necessarily - in Electron you have multiple documents, and being able to specify which DOM that should be used is useful.

Reply:
Ah, TIL!